aspnetcore/signalr/javascript-client/sample/wwwroot/js/chat.js

```diff
@@ -25,4 +25,19 @@ document.getElementById("sendButton").addEventListener("click", event => {
     event.preventDefault();
 });
 
-connection.start().catch(err => console.error(err.toString()));
\ No newline at end of file
+connection.start().catch(err => console.error(err.toString()));
```

Comment:
I noticed there's a *chat.ts* file in this sample app. Thoughts on modifying that instead and letting TS generate this JS file?

Reply:
i have no problem with that but....

> blush

i don't know how to do that..